src/main.js

```diff
@@ -8,16 +6,18 @@ const createElement =(tagName, className)=>{
     return elem;
 }
 
-const render = (data)=>{
-    data.forEach((item)=>{
+const render = async()=>{
+    const url = 'https://fe-student-api.herokuapp.com/api/hotels/popular';
```

Comment:
I recommend to get data by a separate request
If error??? what about to handle errors
```
function getData(url) {
  //request
 ...
  return data;
}
```

Reply:
Split for 2 separate functions: render() and getData().